backends/generators/Go/inst.go

```diff
@@ -0,0 +1,2637 @@
+// Code generated by go_generator.py; DO NOT EDIT.
+package riscv
+
+import "cmd/internal/obj"
+
+type inst struct {
+	opcode uint32
+	funct3 uint32
+	rs1    uint32
+	rs2    uint32
+    csr    int64
```

Comment:
The indentation here is weird.  It doesn't matter too much as we will always format this file before adding to the go source tree, but it's easy to fix in the python file.